### PR TITLE
Add a README for the alpha taxonomy bulk import scripts

### PR DESCRIPTION
**Pull request by mobaig**

### CODE REVIEW

**Comment by tijmenb** [comment]
:+1: great!

**Comment by tijmenb** [comment]
I do suspect we can combine a few steps of the readme in a rake task, is that right?

File: lib/alpha_taxonomy/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  +-------------------+-----------------+
+  ```
+
+0. Make the sheet public on Google Drive in TSV format. Its public URL should
+   look something like this:
+   `https://docs.google.com/spreadsheets/d/19GhkAQ9VEmsiPeoHbrz9Q-nTnbtLxC2kkD6szoGGam0/edit#gid=1102496302`.
+   The `19Ghka...` part is the `key` and the param at the end is the `gid`.
+0. In the environment in which you're running the import, set an environment
+   variable `TAXON_SHEETS` with values in the form
+   `name-of-sheet,the-key,the-gid,name-of-some-other-sheet,its-key,its-gid`.
```
Comment (jackscotti) [inline lib/alpha_taxonomy/README.md]: gid == Google ID?

Comment (mobaig) [inline lib/alpha_taxonomy/README.md]: Yes, I'd assume so.

**Comment by mobaig** [comment]
@tijmenb re: Rake task - yes, definitely. I've been running them separately while developing and improving each stage but it would be useful to codify the run-order in a task. Will add in a followup.

**Comment by tijmenb** [comment]
:+1:

**Pull request merged**